src/plugins/Url/index.js

```diff
@@ -158,7 +162,11 @@ module.exports = class Url extends Plugin {
         if (item.kind === 'string' && item.type === 'text/uri-list') {
           item.getAsString((url) => {
             this.uppy.log(`[URL] Adding file from dropped url: ${url}`)
-            this.addFile(url)
+            try {
+              this.addFile(url)
```

Comment:
This is a different `addFile` function, it doesn't need the try/catch

Reply:
And that’s why we have reviews, thanks! ❤️